openmeter/subscription/price.go

```diff
@@ -0,0 +1,73 @@
+package subscription
```

Comment:
Move these to entity sub pkg please similar to:

https://github.com/openmeterio/openmeter/tree/main/openmeter/app
https://github.com/openmeterio/openmeter/tree/main/openmeter/billing
https://github.com/openmeterio/openmeter/tree/main/openmeter/customer